conf/mozilla-autoconfig.xml

```diff
@@ -3,8 +3,8 @@
     <emailProvider id="PRIMARY_HOSTNAME">
       <domain>PRIMARY_HOSTNAME</domain>
 
-      <displayName>PRIMARY_HOSTNAME (Mail-in-a-Box)</displayName>
-      <displayShortName>PRIMARY_HOSTNAME</displayShortName>
+      <displayName>%EMAILDOMAIN% (Mail-in-a-Box)</displayName>
+      <displayShortName>%EMAILDOMAIN%</displayShortName>
```

Comment:
Please do not use placeholders in the display name. Not all clients implement placeholders in all fields.

Reply:
Can the server insert the correct value here, given that it generates the file and supports server-side placeholders anyways?

Reply:
No, see #954. Mail-in-a-Box uses a single config file for all domains. The `PRIMARY_HOSTNAME` placeholder is replaced at install time, so for example if the user's primary domain is `example.com`, but they also are hosting `example.net` and `example.org`, the `PRIMARY_HOSTNAME` placeholder would be replaced with `box.example.com`: https://github.com/mail-in-a-box/mailinabox/blob/3efd4257b57fe0063ab0f22e9019874e85369a7d/setup/web.sh#L115-L123 This is obviously confusing for the user when they are setting up an `example.net` or `example.org` e-mail address, especially when they have multiple addresses, as they all show as `box.example.com` in their e-mail client's UI.

Reply:
I understand the reason and desire. It would be nice, if this worked.

However, fact is that many AutoConfig clients do not use placeholders in these fields. So, these clients would show the literal `%EMAILDOMAIN%` to the user.

Reply:
I think this change was removed in the last version?

Given that this is a breaking change for some clients, could you please move this to a new PR and reference the new PR number here? Then we can investigate that issue there in the new PR, and the rest here can land already. All the rest is OK, from what I can see.